components/Feedback/ArticleItem.js

```diff
@@ -18,39 +17,37 @@ const styles = {
   }),
   count: css({
     fontFeatureSettings: '"tnum" 1, "kern" 1',
-    color: colors.primary,
     position: 'absolute',
     right: 0
   })
 }
 
-export const NoResultsItem = ({ title }) => (
-  <P style={{ color: colors.disabled }}>{title}</P>
-)
+export const NoResultsItem = ({ title }) => {
+  const [colorScheme] = useColorContext()
+  return <P {...colorScheme.set('color', 'disabled')}>{title}</P>
```

Comment:
Here an additional span is necessary to ensure that color disabled wins:
```suggestion
  return <P><span {...colorScheme.set('color', 'disabled')}>{title}</span></P>
```

Because [`Interaction.P`](https://github.com/orbiting/styleguide/blob/26da62022f3bb887e1b514ad07258cc1d0d97553/src/components/Typography/Interaction.js#L105-L117) does `{...colorScheme.set('color', 'text')}` are clear winner can't be determined and now depends on which css rules get injected last, which is not explicitly defined. It's not a matter of prop order but when the css side effect happens—including during SSR and css text transfer from first SSR render.